src/Evolution/Systems/RadiationTransport/M1Grey/M1Closure.hpp

```diff
@@ -0,0 +1,43 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines functions to compute the M1 closure
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+// Compute the M1 closure and returns the neutrino pressure tensor (in the
```

Comment:
make this a doxygen comment, and describe the algorithm, ideally pointing to a reference